services/web/server/src/simcore_service_webserver/projects/_groups_models.py

```diff
@@ -0,0 +1,18 @@
+import logging
+from datetime import datetime
+
+from models_library.groups import GroupID
+from pydantic import BaseModel, ConfigDict
+
+_logger = logging.getLogger(__name__)
+
+
+class ProjectGroupGetDB(BaseModel):
+    gid: GroupID
+    read: bool
+    write: bool
+    delete: bool
+    created: datetime
+    modified: datetime
+
```

Comment:
beautiful order

Reply:
sarcasm? :-D i not sure how to read this ...